python/tank/platform/engine.py

```diff
@@ -661,6 +644,29 @@ def created_qt_dialogs(self):
         """
         return self.__created_qt_dialogs
 
+    @property
+    def host_info(self):
+        """
+        Returns information about the application hosting this engine.
+        
+        This should be re-implemented in deriving classes to handle the logic 
+        specific to the application the engine is designed for.
+        
+        :returns: A (host application name, release string) tuple.
+        """
+        return (
+            "unknown",
+            "unknown",
+        )
+
+    @property
+    def host_info_string(self):
```

Comment:
should this be filed under "internal methods"? Do we expect/need/want anyone outside of core to ever call this. If no, then we should make it internal?